### PR TITLE
Filter 7 external links to pass ci check

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Filter 7 external links to pass ci check

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: .github/workflows/link-check.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
...
-          args: -c 32 -d /github/workspace -r -x "http://172.*|https://github.com/.*/pull/.*|https://github.com/.*/issues/.*|https://github.com/[^.\n]*$|http://127.0.0.1.*|http://localhost.*|http://.*:.*|https://static.googleusercontent.com/media/research.google.com/.*|http://www.brendangregg.com/perf.html.*|https://blog.csdn.net.*|http://mysql-cocoa.sourceforge.net.*|https://tools.ietf.org.*|http://code.google.com.*|https://spark.apache.org.*|.*.md" .
+          args: -c 32 -d /github/workspace -r -x "http://172.*|https://github.com/.*/pull/.*|https://github.com/.*/issues/.*|https://github.com/[^.\n]*$|http://127.0.0.1.*|http://localhost.*|http://.*:.*|https://static.googleusercontent.com/media/research.google.com/.*|http://www.brendangregg.com/perf.html.*|https://blog.csdn.net.*|http://mysql-cocoa.sourceforge.net.*|https://tools.ietf.org.*|http://code.google.com.*|https://spark.apache.org.*|https://asktug.com/|https://shimo.im/sheets/tTRyydP8Xkdv8yxq/MODOC|http://www.alhem.net/project/mysql/|https://www.figma.com/file/dHQ9B7X4mF1q78NRIzNTs5/tidb-sketch-book-2020|http://www.webgraphviz.com/|http://octave.sourceforge.net/database/index.html|http://gnade.sourceforge.net/|.*.md" .
```
Comment (yikeke) [inline .github/workflows/link-check.yaml:21]:
```suggestion
          args: -c 32 -d /github/workspace -r -x "http://172.*|https://github.com/.*/pull/.*|https://github.com/.*/issues/.*|https://github.com/[^.\n]*$|http://127.0.0.1.*|http://localhost.*|http://.*:.*|https://static.googleusercontent.com/media/research.google.com/.*|http://www.brendangregg.com/perf.html.*|https://blog.csdn.net.*|http://mysql-cocoa.sourceforge.net.*|https://tools.ietf.org.*|http://code.google.com.*|https://spark.apache.org.*|https://asktug.com/*|https://shimo.im/sheets/*|http://www.alhem.net/project/mysql/|http://www.webgraphviz.com/|http://octave.sourceforge.net/database/index.html|http://gnade.sourceforge.net/|.*.md" .
```
I don't have the permission to access https://www.figma.com/file/dHQ9B7X4mF1q78NRIzNTs5/tidb-sketch-book-2020.

Comment (TomShawn) [reply .github/workflows/link-check.yaml:21]: + I searched the repo and did not find this link now. Let's ignore this one.
+ For the shimo.im link, if other shimo.im link is added to the repo and later becomes 404, the link checker will be unable to detect the 404 link.

Comment (yikeke) [reply .github/workflows/link-check.yaml:21]: 👌

**Review by yikeke — APPROVED** [review]
Rest LGTM

**Pull request merged**